src/main/java/pocketpal/frontend/constants/MessageConstants.java

```diff
@@ -24,25 +24,31 @@ public final class MessageConstants {
             + "-d <description>" + NEWLINE
             + "-c <category>" + NEWLINE
             + "-p <price>" + NEWLINE
-            + "See below for examples" + NEWLINE
+            + "*All three options must be specified" + NEWLINE + NEWLINE;
+    public static final String MESSAGE_ADD_COMMAND_EXMPLES = "See below for examples" + NEWLINE
             + "/add -d Apple Macbook Air -p 1300 -c Personal" + NEWLINE
-            + "/add -p 1300 -c Personal -d Apple Macbook Air" + NEWLINE + NEWLINE;
-    public static final String MESSAGE_DELETE_COMMAND = "Delete - Deletes specified entry(s) from your account."
+            + "/add -p 1300 -c Personal -d Apple Macbook Air" + NEWLINE;
+    public static final String MESSAGE_VALID_CATEGORIES = "Valid input categories are:" + NEWLINE
+            + "CLOTHING, ENTERTAINMENT, FOOD," + NEWLINE
+            + "TRANSPORTATION, MEDICAL, PERSONAL," + NEWLINE
+            + "INCOME, UTILITIES, OTHERS" + NEWLINE + NEWLINE;
+    public static final String MESSAGE_VALID_PRICE = "Valid input prices are:" + NEWLINE
+            + "Positive numbers strictly ranging from 0.01 to 999999999.99" + NEWLINE + NEWLINE;
+    public static final String MESSAGE_DELETE_COMMAND = "Delete - Deletes specified entry(s) from your expenditure."
```

Comment:
`expenditure` should be replaced with `account` as per the new naming convention.